server/service/individualServices/RegardApplication.js

```diff
@@ -32,36 +32,39 @@ exports.regardApplication = function (applicationName, releaseNumber, user, xCor
         let timestampOfCurrentRequest = new Date();
         try {
             operationKeyUpdateNotificationService.turnONNotificationChannel(timestampOfCurrentRequest);
+
             const result = await CreateLinkForInquiringOamRecords(applicationName,
                 releaseNumber,
                 user,
                 xCorrelator,
                 traceIndicator,
                 customerJourney,
                 traceIndicatorIncrementer++);
+            let statusForCreateLinkForInquiringServiceRecords = {}
```

Comment:
variable name shall be statusForCreateLinkForInquiring**Oam**Records. Please change it in the definition aswell in the usage.